README.md

```diff
@@ -1,7 +1,41 @@
 # help-center-content
+
 Repository assembling VTEX's Help Center content
 
+## In this repository
+
+You will find the following folders in this repository:
+
+-TBD
+
+## Contributing with Help Center Portal documentation
+
+We're so glad you're here! Thanks for being interested.
+
+### How do I add a download file to my article?
+
+1. Open a branch in the [help-portal-content](https://github.com/vtexdocs/help-center-content) repository.
+2. Copy the file you want to make available for download to the same directory as the article where it will be inserted. For instance, to allow the user to download a spreadsheet within the article **[Payment Provider List](https://github.com/vtexdocs/help-center-content/blob/b454803127cc490362c16921ceee023a5d182409/docs/pt/tutorials/Payments/Payment%20Settings/list-of-payment-providers-by-country.md)**, you need to save it in the directory [vtexdocs/help-center-content/docs/pt/tutorials/Payments/Payment Settings](https://github.com/vtexdocs/help-center-content/docs/pt/tutorials/Payments/Payment_Settings/).
```

Comment:
The directory link doesn't work. Also I think it is more helpful to have the paths than the links in the repository. For instance: `docs/pt/tutorials/Payments/Payment Settings` and `docs/pt/tutorials/Payments/Payment Settings/list-of-payment-providers-by-country.md`.

Reply:
Updated.